app/src/main/java/mozilla/lockbox/presenter/ItemListPresenter.kt

```diff
@@ -80,26 +86,40 @@ class ItemListPresenter(
             .subscribe(this::onMenuItem)
             .addTo(compositeDisposable)
 
+        view.lockNowClick
```

Comment:
awesome!!

if this was me, rather than a function here I've been doing something more like:
```kotlin
.map {
    // convert to appropriate route action here
  }
  .subscribe(dispatcher::dispatch)
```

but that is totally an implementation nit ;) this definitely works!!